src/main/java/com/checkmarx/flow/service/ResultsService.java

```diff
@@ -334,14 +334,14 @@ private boolean requiresCxCustomFields(List<Field> fields) {
         return false;
     }
 
-    public boolean filteredIssuesPresent(ScanResults results) {
+    public boolean filteredSastIssuesPresent(ScanResults results) {
         if(results.getAdditionalDetails()==null) {
             // assuming SCA results only
             return false;
         }
 
-        Map<String, Integer> flow = (Map<String, Integer>) results.getAdditionalDetails().get(Constants.SUMMARY_KEY);
-        return flow == null || !flow.isEmpty();
+        Map<String, Integer> findingCountPerSeverity = (Map<String, Integer>) results.getAdditionalDetails().get(Constants.SUMMARY_KEY);
+        return findingCountPerSeverity == null || !findingCountPerSeverity.isEmpty();
```

Comment:
If e.g. findingCountPerSeverity has a single 'HIGH' key with the value 0, then findingCountPerSeverity.isEmpty() will return false. The result of filteredSastIssuesPresent() will then be 'true'. However, in fact no issues are present.